src/lib.rs

```diff
@@ -923,6 +941,27 @@ mod tests {
         }
     }
 
+    mod write {
+        use std::error::Error;
+        use super::RECS;
+        use super::REC_SIZE;
+        use super::super::*;
+
+        #[test]
+        fn should_write_record() {
+            let mut vec = Vec::new();
+
+            let record = Record::parse(&RECS.as_bytes()[..963]).unwrap();
+
+            match vec.write_record(record.clone()) {
+                Err(why) => panic!("couldn't write file: {}", why.description()),
+                Ok(_) => (),
+            }
+
+            assert_eq!(vec.len(), REC_SIZE as usize);
```

Comment:
I know that implantation is straightforward, but could you assert equality of actual record content?